week 5/README.md

```diff
@@ -109,6 +109,47 @@ for (int i = 0; i < n; ++i)
 
 ### Arrays
 
+The **array** is such a **data structure** that stores a fixed-size sequencial elements of a certain type.
```

Comment:
Some typos here and on the next lines - *sequen**T**ial* for example. It also doesn't make it clear that the array has a fixed size, not its elements. How about *...that stores a fixed number of elements*?